tests/tests.qbs

```diff
@@ -0,0 +1,100 @@
+import qbs
+import qbs.FileInfo
+import qbs.File
+
+Project {
+    name: "QSqlMigrator.tests"
+
+
+    Application {
+        name: "ApiTest"
+        consoleApplication: true
+        type: base.concat("autotest")
+        install: true
+
+        Depends { name: "QSqlMigrator" }
+        Depends { name: "cpp" }
+        Depends { name: "Qt"; submodules: ["core", "sql", "test"] }
+        Depends { name: "SqliteMigrator" }
+
+        Group {
+            name: "api"
+            prefix: "api/"
+            files: [
+                "tst_ApiTest.cpp",
+                "M20132201_180943_CreateCars.cpp",
+                "M20132201_175827_CreateAddresses.cpp",
+                "M20131501_191807_CreateUsers.cpp",
+                "M20133001_164323_AddUsers.cpp",
+                "SqliteAddUser.cpp",
+                "M20132201_180943_CreateCars.h",
+                "M20132201_175827_CreateAddresses.h",
+                "M20131501_191807_CreateUsers.h",
+                "M20133001_164323_AddUsers.h",
+                "SqliteAddUser.h",
+                "ApiConfig.h",
+                "ApiConfig.h.example",
+            ]
+        }
+
+        files: [
+            "BasicTest/BasicTest.h",
+            "BasicTest/BasicTest.cpp"
+        ]
+    }
+
+    Application {
+        name: "mysqlTest"
+
+
+        Depends { name: "MysqlMigrator" }
+        Depends { name: "QSqlMigrator" }
+        Depends { name: "SqliteMigrator" }
+        Depends { name: "cpp" }
+        Depends { name: "Qt"; submodules: ["core", "sql", "test"] }
```

Comment:
Only "test" is necessary here. "core" and "sql" are already provided through exports of dependencies.